packages/component-submission/server/use-cases/supportingFiles.js

```diff
@@ -48,21 +48,23 @@ class SupportingFiles {
   }
 
   async removeAll() {
-    let manuscript = await ManuscriptModel.find(this.manuscriptId, this.userId)
-    const filesWithoutSupporting = manuscript.files.filter(
-      file => file.type !== 'SUPPORTING_FILE',
-    )
+    const manuscript = new Manuscript(config, this.userId, this.storage)
```

Comment:
I cannot see where the `manuscript` variable is used ... we do need to check the user has access to this manuscript tho before trying to remove it ... i.e. I think we should re-instate line 53